setup.py

```diff
@@ -24,25 +24,21 @@ def get_version(filename):
 setup(
     name='django-cors-headers',
     version=version,
-    description=(
-        'django-cors-headers is a Django application for handling the server '
-        'headers required for Cross-Origin Resource Sharing (CORS).'
-    ),
+    description='A Django App that adds CORS (Cross-Origin'
+                ' Resource Sharing) headers to responses.',
```

Comment:
When you make a pull request to a project please don't modify unrelated bits of code to suit your preferred style, instead try to fit into the style of the project as you find it :)